src/main/java/edu/harvard/iq/dataverse/api/DatasetFieldServiceApi.java

```diff
@@ -256,6 +260,10 @@ public Response loadDatasetFields(File file) {
                         case "#controlledVocabulary":
                             header = HeaderType.CONTROLLEDVOCABULARY;
                             break;
+                        case "#controlledVocabularyExt":
```

Comment:
Can you please document `#controlledVocabularyExt` in source/admin/metadatacustomization.rst?

Reply:
"#controlledVocabularyExt" removed